spec.html

```diff
@@ -7649,58 +7649,42 @@ <h1>Jobs and Job Queues</h1>
-    <p>Each agent has its own set of named Job Queues.  All references to a named job queue in this specification denote the named job queue of the surrounding agent.</p>
-    <emu-table id="table-26" caption="Required Job Queues">
-      <table>
-        <tbody>
-        <tr>
-          <th>
-            Name
-          </th>
-          <th>
-            Purpose
-          </th>
-        </tr>
-        <tr>
-          <td>
-            ScriptJobs
-          </td>
-          <td>
-            Jobs that validate and evaluate ECMAScript |Script| and |Module| source text. See clauses 10 and 15.
-          </td>
-        </tr>
-        <tr>
-          <td>
-            PromiseJobs
-          </td>
-          <td>
-            Jobs that are responses to the settlement of a Promise (see <emu-xref href="#sec-promise-objects"></emu-xref>).
-          </td>
-        </tr>
-        </tbody>
-      </table>
-    </emu-table>
-    <p>A request for the future execution of a Job is made by enqueueing, on a Job Queue, a PendingJob record that includes a Job abstract operation name and any necessary argument values. When there is no running execution context and the execution context stack is empty, the ECMAScript implementation removes the first PendingJob from a Job Queue and uses the information contained in it to create an execution context and starts execution of the associated Job abstract operation.</p>
-    <p>The PendingJob records from a single Job Queue are always initiated in FIFO order. This specification does not define the order in which multiple Job Queues are serviced. An ECMAScript implementation may interweave the FIFO evaluation of the PendingJob records of a Job Queue with the evaluation of the PendingJob records of one or more other Job Queues. An implementation must define what occurs when there are no running execution context and all Job Queues are empty.</p>
-    <emu-note>
-      <p>Typically an ECMAScript implementation will have its Job Queues pre-initialized with at least one PendingJob and one of those Jobs will be the first to be executed. An implementation might choose to free all resources and terminate if the current Job completes and all Job Queues are empty. Alternatively, it might choose to wait for a some implementation specific agent or mechanism to enqueue new PendingJob requests.</p>
-    </emu-note>
-    <p>The following abstract operations are used to create and manage Jobs and Job Queues:</p>
+    <p>Jobs are scheduled for execution by ECMAScript host environments with abstract operations. The ECMAScript specification describes the host hook HostEnqueuePromiseJob to schedule one job; host environments may define additional abstract operations which schedule jobs. Such operations accept a PendingJob Record _pending_ as a parameter. These abstract operations schedule the PendingJob to be performed at some future time. Their implementations must conform to the following requirements:</p>
```

Comment:
```suggestion
    <p>Jobs are scheduled for execution by ECMAScript host environments with abstract operations. The ECMAScript specification describes the host hook HostEnqueuePromiseJob to schedule one job; host environments may define additional abstract operations which schedule jobs. Such operations accept a PendingJob Record as a parameter. These abstract operations schedule the PendingJob to be performed at some future time. Their implementations must conform to the following requirements:</p>
```

Remove unused reference.

Reply:
GitHub's "commit suggestion" feature doesn't seem to be working. Feel free to commit this suggestion with the PR.